File: python/nivel 3 listas textos/16.py
"""
Ejercicio 16
Dado un arreglo, devolver solo los números pares
"""
numeros  = [4,7,10,15,22,33,8,9]

pares = []

for numero in numeros:
  if numero % 2 == 0:
    pares.append(numero)

print("Numeros Pares: ", pares)

File: javascript/nivel 3 listas textos/16.js
/*
Ejercicio 16
Dado un arreglo, devolver solo los números pares
*/
let numeros = [4,7,10,15,22,33,8,9];

let pares = [];

for (let i = 0; i < numeros.length; i++) {
  if(numeros[i] % 2 === 0){
    pares.push(numeros[i]);
  }
}

console.log("Numeros Pares: ", pares);